pkg/component/component.go

```diff
@@ -57,13 +73,12 @@ func CreateFromPath(client *occlient.Client, name string, ctype string, path str
 	annotations := map[string]string{componentSourceURLAnnotation: sourceURL.String()}
 	annotations[componentSourceTypeAnnotation] = sourceType
 
-	err := client.NewAppS2I(name, ctype, "", labels, annotations)
+	err := client.NewPseudoVPS(name, ctype, labels, annotations)
```

Comment:
VPS? I think you mean PVC?

Maybe "MockPVC" or "FakePVC" would be better since pseudo is hard to type / non-english speakers may be confused.